libwhich.c

```diff
@@ -211,6 +211,7 @@ struct vector_t dllist()
     return dynamic_libraries;
 }
 
+#ifdef dlinfo
```

Comment:
it seems unlikely this is a macro
```suggestion
#if defined(__linux__) || defined(__FreeBSD__) # Use `dlinfo` API, when supported
```

Reply:
Fair point; the fix seems sane to me.